elements/pf-code-block/demo/pf-code-block.html

```diff
@@ -13,13 +13,13 @@ <h2>Basic</h2>
       connectionConfig:
       url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs
     </script>
-    <pfe-clipboard slot="actions" copy-from="#code"></pfe-clipboard>
-  </pfe-code-block>
+    <pf-clipboard slot="actions" copy-from="#code"></pf-clipboard>
```

Comment:
`<pf-clipboard ...></pf-clipboard>` should be `<pf-clipboard-copy ...></pf-clipboard-copy>`